pkg/controller/vitesskeyspace/reconcile_resharding.go

```diff
@@ -13,25 +31,73 @@ import (
 	"vitess.io/vitess/go/vt/wrangler"
 )
 
-func (r *ReconcileVitessKeyspace) reconcileResharding(ctx context.Context, vtk *planetscalev2.VitessKeyspace) error {
-	wr, ts, err := newWrangler(ctx, vtk.Spec.GlobalLockserver)
+const (
+	maxSafeVReplicationLag = 10
+)
+
+func (r *reconcileHandler) reconcileResharding(ctx context.Context) error {
+	wr, ts, err := newWrangler(ctx, r.vtk.Spec.GlobalLockserver)
```

Comment:
Do we still need this if we initialize a wrangler with `tsInit()` that lives on the `reconcileHandler` object? Or is that for a different purpose?

Reply:
Still working on it - but already removed in the code that's local. Will push up soon.

Reply:
Done.